### PR TITLE
[CCTRI-2422] - Add warning and 403 error handling 

### PR DESCRIPTION
**Pull request by ytsek**
- Add warning and 403 error handling
- Update auth error handling
- flask update

### CODE REVIEW

**Review by MaxMhonc — DISMISSED** [review]

**Review by AnastasiiaRozlyvan — DISMISSED** [review]

File: code/requirements.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+Flask==2.0.1
 marshmallow==3.11.1
 requests==2.25.1
 flake8==3.9.0
 coverage==5.5.0
 pytest==6.2.2
 cryptography==3.3.2
 pyjwt[crypto]==2.0.1
-Werkzeug==1.0.1
+freezegun==1.1.0
```
Comment (TarasMaliuchenko) [inline code/requirements.txt:9]: Where it is used?

Comment (ytsek) [reply code/requirements.txt:9]: It's used in tests for timestamp mocking.

Comment (TarasMaliuchenko) [reply code/requirements.txt:9]: And why it is this date`2021-01-14` ?

Comment (ytsek) [reply code/requirements.txt:9]: It's arbitrary chosen date.

Comment (TarasMaliuchenko) [reply code/requirements.txt:9]: Is it possible that in case of receive another date our module will crash?

Comment (ytsek) [reply code/requirements.txt:9]: To crash this module you need to change value in decorator or in test function body.

**Review by TarasMaliuchenko — APPROVED** [review]

**Review by AnastasiiaRozlyvan — APPROVED** [review]

**Pull request merged**